providers/base/bin/mouse_keyboard.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python3
+#
+# This file is part of Checkbox.
+#
+# Copyright 2016 Canonical Ltd.
+#
+# Authors:
+#   Gabriel Chen <gabriel.chen@canonical.com>
+#
+# Checkbox is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3,
+# as published by the Free Software Foundation.
+#
+# Checkbox is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Checkbox.  If not, see <http://www.gnu.org/licenses/>.
+"""mouse_key_random utility."""
+
+
+from evdev import UInput, ecodes as e
+import time
+import random
```

Comment:
```suggestion
import time
import random
from evdev import UInput, ecodes as e
```

Importing stdlib stuff first.